core/modules/util/ConfigStore.cc

```diff
@@ -51,25 +53,29 @@ namespace qserv {
 namespace util {
 
 
-void ConfigStore::parseFile(std::string const& configFilePath) {
+std::map<std::string, std::string> const ConfigStore::_parseIniFile(std::string const& configFilePath) {
 
     // read it into a ptree
     boost::property_tree::ptree pt;
     boost::property_tree::ini_parser::read_ini(configFilePath, pt);
 
+    std::map<std::string, std::string> configMap;
+
     // flatten
     for (auto& sectionPair: pt) {
         auto& section = sectionPair.first;
         for (auto& itemPair: sectionPair.second) {
             auto& item = itemPair.first;
             auto& value = itemPair.second.data();
-            _configMap.insert(std::make_pair(section + "." + item, value));
+            configMap.insert(std::make_pair(section + "." + item, value));
         }
     }
+
+    return configMap;
 }
 
-std::string ConfigStore::get(std::string const& key) const {
-    StringMap::const_iterator i = _configMap.find(key);
+std::string ConfigStore::getString(std::string const& key) const {
+    std::map<std::string, std::string>::const_iterator i = _configMap.find(key);
```

Comment:
`_configMap` is const, so the iterator you will get from `find()` will be `const_iterator`